crypto/src/main/kotlin/web5/sdk/crypto/KeyManager.kt

```diff
@@ -14,6 +15,8 @@ import web5.sdk.crypto.jwk.Jwk
  */
 public interface KeyManager {
 
+  fun getCore(): KeyManagerCore
```

Comment:
this is going to be a common practice where we expose new functions to return the "core" type for the given type